### ISSUE TITLE
Confirm if multiple text links to collections on /collections fails accessibility checks

### ISSUE THREAD

**Issue by michael-wynne-wsu** [MukurtuCMS/Mukurtu-CMS#655]
If it's a warning we can leave as it, otherwise we need to remove one (probably the title).
